### ISSUE TITLE
Missing first level heading

### ISSUE THREAD

**Issue by brianteeman** [joomla/cassiopeia#124]
### Steps to reproduce the issue
Install current development branch and sample data
go to the About page

### Expected result
Good semantic markup and no accessibility errors

### Actual result
No H1 on the page

.
**Why It Matters**
Headings facilitate page navigation for users of many assistive technologies. They also provide semantic and visual meaning and structure to the document. A first level heading (`<h1>`) should be present on nearly all pages. It should contain the most important heading on the page (generally the document title).

**Standards and Guidelines**
1.3.1 Info and Relationships (Level A)
2.4.6 Headings and Labels (Level AA)

**Comment by drmenzelit** [comment]
@chmst it this related with sample data?

**Comment by richard67** [comment]
> @chmst it this related with sample data?

I think it is.

**Comment by chmst** [comment]
Yes and no. 
It is true that a Website should contain at least one H1. But this is in the responsibility of the webdesigner.  Where and how he adds this. So the question is: are sample data just for demonstration of some params and features or is the purpose also to give a fully accessible site. 
In a real site I use template overrides for setting of h1 in article header. For sample data this is not the richt way. So we can add a banner module which sets a H1 in single article sites.

**Comment by brianteeman** [comment]
The demonstration data is what we will be judged on when someone evaluates joomla for the first time.

**Comment by chmst** [comment]
I add the joomla-cms label, ti is not the task of a template, but made as much as possible
